bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -734,10 +734,15 @@ Toggle to turn on no competition mode (only relevant if FATES is being used).
 Toggle to turn on FATES satellite phenology mode (only relevant if FATES is being used).                        
 </entry>
 
-<entry id="use_fates_logging" type="logical" category="physics"
-        group="clm_inparm" valid_values="" value=".false.">
-Toggle to turn on the logging module
-(Only relevant if FATES is on)
+<entry id="fates_harvest_mode" type="char*13" category="physics"
+        group="clm_inparm" valid_values="no_harvest,event_code,surfdata_file,luhdata_area,luhdata_mass">
+Set FATES harvesting mode by setting fates_harvest_mode to a valid string option.
+Allowed values are:
+ no_harvest: no fates harvesting of any kind
+ event_code: fates logging via fates logging event codes (see fates parameter file) only
+ surfdata_file: fates harvest driven by CLM landuse timeseries data (dynHarvestMod)
+ luhdata_area: fates harvest driven by LUH2 raw harvest data, area-based (dynFATESLandUseChangeMod)
+ luhdata_mass: fates harvest driven by LUH2 raw harvest data, mass-based (dynFATESLandUseChangeMod)
```

Comment:
@glemieux and I talked about each of the modes and how they are used. Some are more important than others, and I got the impression that two of them might be deprecated in the future (event_code and landusepftdat_file). So maybe have a note about them being older features?

Reply:
addressed via 31803d9

Reply:
That's a good question about `event_code`.  I will start up a discussion on the fates repo to talk about it for a future possible update.  The `event_code` option is a discrete option that we want to keep, but the way in which we engage it currently could possibly be better explicated and perhaps even segregated.